src/main/java/http/Router.java

```diff
@@ -21,6 +22,7 @@ public Router() {
         routeHashMap.put("/text_response", new TextResponse());
         routeHashMap.put("/html_response", new HTMLResponse());
         routeHashMap.put("/json_response", new JSONResponse());
+        routeHashMap.put("/xml_response", new XMLResponse());
     }
 
     public Response getResponse(String method, String path, String body) {
```

Comment:
It would read better if you created a class for `NotFound` route:

so 
```return new ResponseBuilder()
                .withStatusCode(HTTPStatusCodes._404.getCode())
                .withHeader("")
                .withBody("")
                .build();
    }`
```

reads something like: 
```
      else {
       Route route = new NotFound();
            return route.response(header, body);
        }
```